src/utils/api.js

```diff
@@ -13,19 +13,26 @@ export default {
   oauthToken: '',
   spaceId: null,
   region: '',
+  client: false,
 
   getClient () {
     const { region } = creds.get()
 
     try {
-      return new Storyblok({
-        accessToken: this.accessToken,
-        oauthToken: this.oauthToken,
-        region: this.region,
-        headers: {
-          ...DEFAULT_AGENT
-        }
-      }, this.apiSwitcher(region))
+      if (this.client === false) {
```

Comment:
```suggestion
      if (!this.client) {
```

Reply:
Hola @alvarosabu , thank you for your feedback. I adapted the code according to your feedback